core/testsuite/src/test/java/com/blazebit/persistence/testsuite/KeysetPaginationTest.java

```diff
@@ -39,14 +41,15 @@
 
 /**
  * @author Christian Beikov
+ * @author Moritz Becker
  * @since 1.0
  */
 public class KeysetPaginationTest extends AbstractCoreTest {
 
     private Person o4;
 
-    @Override
-    public void setUpOnce() {
+    @Before
```

Comment:
In this case I will need to extract the `keysetPaginationPage1EdgeCaseFallback` into a separate test class that extends the abstract keyset pagination test class since it needs to be able to modify the database contents. Do you comply?